_includes/payment-state.md

```diff
@@ -0,0 +1,31 @@
+{% assign api_resource = include.api_resource | default: "`paymentorder`" %}
+{% assign transaction = include.transaction | default: "`authorization`" %}
```

Comment:
Since `transaction` is never sent in via the `include`, can't it just be removed and hard coded in the text to always say `authorization`? If not, you need to pass it in. Also, remove the formatting from the variable assignment and add it inside the text where the variable is used instead.

```suggestion
{% assign transaction = include.transaction | default: "authorization" %}
```